setup.cfg

```diff
@@ -2,7 +2,7 @@
 description-file = README.md
 
 [bdist_rpm]
-requires=python python-six
+requires=python3-six
```

Comment:
`six` isn't a dependency on this package. Why is this needed?

Reply:
You are right, don't see why it should be there - removed it!